src/core/server/base.js

```diff
@@ -201,7 +202,7 @@ function baseServer(
   // Following the ops monitoring convention, return version info at this URL.
```

Comment:
While you're in here could you mention that this is specifically a Dockerflow convention? And link to https://github.com/mozilla-services/Dockerflow